app/main/views/suppliers.py

```diff
@@ -361,3 +359,66 @@ def set_selection_questions(supplier_id, framework_slug):
         abort(400, "Database Error: {}".format(e))
 
     return jsonify(selectionAnswers=supplier_framework.serialize()), status_code
+
+
+@main.route('/suppliers/<supplier_id>/frameworks/interest', methods=['GET'])
+def get_registered_frameworks(supplier_id):
+    supplier_frameworks = SupplierFramework.query.filter(
+        SupplierFramework.supplier_id == supplier_id
+    ).all()
+    slugs = []
+    for framework in supplier_frameworks:
+        framework = Framework.query.filter(
+            Framework.id == framework.framework_id
+        ).first()
+        slugs.append(framework.slug)
+
+    return jsonify(frameworks=slugs)
```

Comment:
I'm not sure - yes I guess it would make sense to have the whole framework to save on extra API calls to get the framework details.  I'll change this.